packages/bpk-docs/src/constants/redirect-routes.js

```diff
@@ -60,7 +60,8 @@ export default {
   '/components/native/badge': ROUTES.BADGE,
   '/components/native/banner-alerts': ROUTES.BANNER_ALERT,
   '/components/native/button': ROUTES.BUTTON,
-  '/components/native/button-link': ROUTES.BUTTON_LINK,
```

Comment:
Whats happening here?

Reply:
`/components/native/button-link` goes to `/components/link`
same for 
'/components/button-link' as we call em just link across the board, the component name in RN is misleading